Release v1.19.1

planarvoid opened the pull request:
This PR points to changes in Aztec. These changes don't touch any existing interfaces and only add new functionality so they shouldn't break anything.
Aztec changes - https://github.com/wordpress-mobile/AztecEditor-Android/pull/887/commits/807062ab67edb46a777883aedc107fc80bb13659

We only want to merge this once the Aztec version is released. 
Did I update the GB version correctly @hypest ?

**Related PRs**
- Add content watcher for all the changes to AztecText - https://github.com/wordpress-mobile/AztecEditor-Android/pull/887
- WPAndroid PR - https://github.com/wordpress-mobile/WordPress-Android/pull/11065

**Test Plan** 
- There is nothing to tests since this is only a new interface not used anywhere in the code yet

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

marecar3 commented:
Hey @planarvoid 👋 
Maybe I can help with all your questions.

> Did I update the GB version correctly @hypest?

Yes, it seems that it's correct.

Regarding this PR I have some comments:

- We usually set the title of the PR to match release version: e.g. Release v1.19.1
- Also, we set the label `release-process`
- And usually, branch name should match the version of the release and it should have prefix release

You can see a nice pattern here: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1627

Note: We should also reference proper WPAndroid PR which will update WPAndroid app with the new version of aztec and especially a new version of the gb-mobile.

Hey @etoledom, not sure if we need to update WPiOS as with `1.19.1` we are just updating the android aztec, what's your impression? Thanks!

planarvoid commented:
Hey @marecar3,
thanks for the check, I've renamed the PR, set the label and added a link to the Aztec changes. I don't have a PR for the WPAndroid merge of Gutenberg release since I don't know how that's done. Do you think you could help me with that? I also don't think it's very important to have this release in WPAndroid or WPiOS since there are no changes to the current logic.

hypest commented:
The Android device tests are failing (on da418661) because of https://github.com/wordpress-mobile/gutenberg-mobile/issues/1703, which is not related to the changes in this PR.
